tests/lib/rules/padding-around-describe-blocks.spec.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * @fileoverview Enforces single line padding around describe blocks
+ * @author Dan Green-Leipciger
```

Comment:
I dunno if you have a preference on `@author` or not with this stuff. Personally, I'm fine leaving me off of stuff. Lemme know... this was in the one that I copy/pasted from.

Reply:
I'm easy. You can put your name there. You can also just remove it. Up to you

Reply:
Let's make up a third person.